packages/react-app/src/app/app.tsx

```diff
@@ -9,7 +9,7 @@ const App = () => (
   <Routes>
     <Route path="/" element={<AuthRoute />}>
       <Route index element={<Manage />} />
-      <Route path="edit" element={<EditGiveaway />} />
+      <Route path="/edit/:id" element={<EditGiveaway />} />
```

Comment:
Maybe we should create a `/giveaways` route and have this one inside that since we will also have routes to edit locations